packages/post-rector/src/Application/PostFileProcessor.php

```diff
@@ -28,9 +29,11 @@ public function __construct(array $postRectors)
      * @param Node[] $nodes
      * @return Node[]
      */
-    public function traverse(array $nodes): array
+    public function traverse(array $nodes, ?SmartFileInfo $smartFileInfo = null): array
     {
         foreach ($this->postRectors as $postRector) {
+            $postRector->addCurrentSmartFileInfo($smartFileInfo);
```

Comment:
`traverse()` contract should be 1:1 to original `NodeTraverser`, to prevent cofussion

`CurrentFileInfoProvider` services should be used instead.